compiler/ballerina-lang/src/main/java/io/ballerina/projects/directory/BuildProject.java

```diff
@@ -136,7 +136,9 @@ public DocumentId documentId(Path file) {
             Path parent = Optional.of(file.toAbsolutePath().getParent()).get();
             for (ModuleId moduleId : this.currentPackage().moduleIds()) {
                 String moduleDirName;
-                if (moduleId.moduleName().contains(DOT)) {
+                // Check for the module name contains a dot and not being the default module
+                if (moduleId.moduleName().contains(DOT)
```

Comment:
We can remove the check for `DOT` here and keep just the condition to check if it is a non-default module.

Reply:
Done